scripts/build.py

```diff
@@ -410,6 +410,7 @@ def __init__(self,
                  dynamo_home = None,
                  target_platform = None,
                  skip_tests = False,
+                 uncompressed = False,
```

Comment:
Given the "global" context here, I think it should be bob_use_compression (or something with bob at least)

Reply:
Renamed to `keep-bob-uncompressed`:
> ./scripts/build.py build_bob --skip-tests --keep-bob-uncompressed